src/modules/main/types/experts.ts

```diff
@@ -0,0 +1,39 @@
+export interface IExperts {
+    userId: number,
+    emailId: string,
+    password: string,
+    status: ExpertStatus,
+    firstName: string,
+    secondName: string,
+    qualification: string,
+    phone: string,
+    bio: string,
+    createdAt: Date,
+}
+
+enum ExpertStatus {
+    'new',
+  'active',
+  'deleted'
+}
+
+export interface IExpertInstitution{
```

Comment:
looks like this is obsolete,
 we can just have another property on the experts interface if needed